src/EditorFeatures/CSharpTest2/Recommendations/VarKeywordRecommenderTests.cs

```diff
@@ -80,16 +80,16 @@ public VarKeywordRecommenderTests()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotInCastType()
+        public async Task TestInCastType()
         {
-            await VerifyAbsenceAsync(AddInsideMethod(
+            await VerifyKeywordAsync(AddInsideMethod(
 @"var str = (($$"));
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotInCastType2()
+        public async Task TestInCastType2()
```

Comment:
I'd name this (or add a comment) to indicate we want this becaue there might be a deconstruction going on.